File: introduction/ITP1_6_A-Reversing Numbers.cpp
#include <iostream>
using namespace std;

int main() {
  int n, i;
  cin >> n;
  int arr[n];
  for(i = n; --i >= 0;) cin >> arr[i];
  for(; ++i < n;) cout << arr[i] << (i != n-1? ' ':'\n');
  return 0;
}

File: introduction/ITP1_6_A-Reversing Numbers.js
process.stdin.resume();
process.stdin.setEncoding('utf8');

var input_string = '';

process.stdin.on('data', function(chunk) {
  input_string += chunk;
});

process.stdin.on('end', function() {
  var lines = input_string.split('\n');
  var arr = lines[1].split(' ').reverse();
  console.log(arr.join(' '));
});
